### PR TITLE
Feature/plant page

### PR DESCRIPTION
**Pull request by jedrzejratajczak**

### CODE REVIEW

**Review by kami3la — APPROVED** [review]

**Review by adax10 — APPROVED** [review]

**Comment by brzeczkowskaw** [comment]
image

- description of plant seems too tight (when text is longer fe. for Epipremnum ), I think there should be more spaces, additinaly, conditioners and subsoil are centered, 
- I think photo and description of plant should be centered, 
- on smaller screen footer is on the comments section, not on the bottom of screen. Perhaps adding scroll to container would help.

File: src/components/PlantInfo/PlantInfo.js [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+            <span>Spraying: </span>
+            <span>{plantDetails.spraying}</span>
+          </p>
+          <p>
+            <span>Toxicity: </span>
+            <span>{plantDetails.toxicity?.human ? 'Yes' : 'No'}</span>
+          </p>
+          <p>
+            <span>Safe for domestic animals: </span>
+            <span>{plantDetails.toxicity?.animal ? 'Yes' : 'No'}</span>
```
Comment (kami3la) [inline src/components/PlantInfo/PlantInfo.js]: And if toxicity is true then it isn't save for animals

Comment (adax10) [reply src/components/PlantInfo/PlantInfo.js]: I would also change to start with a lowercase letter because now it's different from the rest

**Comment by jedrzejratajczak** [comment]
@brzeczkowskaw @KonradMierzejewski @Suegro24 check the branch and approve if everything alright

**Review by brzeczkowskaw — APPROVED** [review]

**Pull request merged**